### PR TITLE
GCE engine

### PR DESCRIPTION
**Pull request by tomkukral**
Test deployment 

```
./devenv.py
./test_gke.py
```

cc @pupapaik

### CODE REVIEW

File: kqueen/engines/gce.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+
+class GceEngine(BaseEngine):
+    """
+    Google Container Engine
+    """
+    name = 'gce'
+    verbose_name = 'gce'
+#    client_id = config.get('GCE_CLIENT_ID')
+#    client_secret = config.get('GCE_CLIENT_SECRET')
```
Comment (tomkukral) [inline kqueen/engines/gce.py]: Please delete these comments unless these config options are necessary.

File: requirements.txt [hunk 2]
```diff
@@ -21,3 +22,8 @@ pytest-ordering
 sphinx
 sphinx-autobuild
 sphinx_rtd_theme
+
+#GCE
```
Comment (tomkukral) [inline requirements.txt]: Please add these dependencies to `setup.py` as well. (section `install_requires`, alphabetic sort)

File: setup.cfg [hunk 3]
```diff
@@ -15,11 +15,13 @@ exclude =
 	./scripts,
 	__pycache__,
 	devenv.py
+        test_gke.py
```
Comment (tomkukral) [inline setup.cfg]: This will ignore `test_gke.py` only for `flake8` - coding style checks. Not needed imho.

**Comment by pupapaik** [comment]
@tomkukral please do review

**Pull request merged**